stream/src/main/scala/org/apache/pekko/stream/impl/fusing/LazyFutureSource.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is part of the Apache Pekko project, which was derived from Akka.
```

Comment:
I think this is the wrong header.

@He-Pin did you write this code yourself or did you copy it from Akka. If you wrote it and did an Akka PR, you still own the copyright on the code.

I think the code should have an ASL header like this file:

https://github.com/apache/incubator-pekko/blob/main/project/AddMetaInfLicenseFiles.scala

Reply:
I copied  it from another file, will update.

Reply:
that special header saying `derived from Akka` is only for code that we took from Akka